alma/lib/Validators/PaymentValidation.php

```diff
@@ -312,6 +322,9 @@ public function validatePayment($almaPaymentId)
      * @param $cartId
      *
      * @return \OrderCore|null
+     *
+     * @throws \PrestaShopDatabaseException
+     * @throws \PrestaShopException
      */
     private function getOrderByCartId($cartId)
```

Comment:
New throws in doc bloc , no try catch in  usage.

Reply:
@Francois-Gomis What do you mean ? Remove the doc bloc about throws ?